cpp/CMakeLists.txt

```
@@ -205,10 +205,10 @@ add_custom_target(python_cffi DEPENDS cudf PYTHON_CFFI)
 install(TARGETS cudf rmm
         DESTINATION lib)
 
-install(FILES ${CMAKE_BINARY_DIR}/../include/cudf.h
+install(FILES include/cudf.h
```

I would be more comfortable with a path that is clearly relative to the CMakeLists.txt location:
```suggestion
install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/include/cudf.h
```

---

Well, files paths are by default relative to the current source directory; but - sure, why not.

---

Done.